scripts/delta.sbatch.sh

```diff
@@ -33,11 +33,11 @@ srun_cmd="srun -N $nnodes -n $nproc"
 # See
 # https://mirgecom.readthedocs.io/en/latest/running.html#avoiding-overheads-due-to-caching-of-kernels
 # on why this is important
-export XDG_CACHE_HOME_ROOT="/tmp/$USER/xdg-scratch/rank"
+export XDG_CACHE_HOME_ROOT="$(pwd)/xdg-cache/rank"
```

Comment:
Small request: Could we make a parent directory for all the cache dirs (preferably hidden)? `.mirge-cache`?

Reply:
I like this idea.  I would do it something like in the `lassen_parallel_spawner.sh` by defaulting `MIRGE_CACHE_ROOT=${MIRGE_CACHE_ROOT:-"$(pwd)/.mirge-cache"`.    Does that sound OK @majosm ?

Reply:
Done in [dca839d](https://github.com/illinois-ceesd/mirgecom/pull/1017/commits/dca839dd58f4bff354d907f55c6226c536001c16).